recipes-support/notify-led/files/LICENSE

```diff
@@ -0,0 +1,54 @@
+Apache License
```

Comment:
Why are we adding licenses everywhere? meta layer already has a license that's applied to everything inside.